test/reputation-system/happy-paths.js

```diff
@@ -217,7 +217,9 @@ contract("Reputation Mining - happy paths", accounts => {
       await advanceMiningCycleNoContest({ colonyNetwork, client: goodClient, test: this });
     });
 
-    it("should be able to process a large reputation update log even if it's using the solidity patricia tree", async () => {
+    // eslint-disable-next-line max-len
+    it("should be able to process a large reputation update log even if it's using the solidity patricia tree", async function largeReputationLogTestSolidity() {
```

Comment:
Where is `largeReputationLogTestSolidity` defined? I can't find it anywhere..

Reply:
This is the definition. It's not used anywhere, but because we need to set the timeout to be longer, we can't use arrow notation (which causes us to lose the definition of `this`) and if we don't name it, we get [eslint shouting at us](https://eslint.org/docs/rules/func-names).